wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -1,178 +1,921 @@
+[GPG guide by Duncan]: http://mrexodia.cf/guide/2016/05/30/Github-Gpg "mrexodia blog post on GitHub GPG by Duncan"
+
+<!-- GH Help -->
+[GitHub Help]: https://help.github.com/ "GitHub Help"
+[GH account]: https://help.github.com/articles/signing-up-for-a-new-github-account/ "GitHub Help in Signing up an account"
+[GFM]: https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help on Writing on GitHub"
+[GH Help feature branch]: https://help.github.com/articles/what-is-a-good-git-workflow/ "GitHub Help on Git workflow"
+[GH Guide feature branch]: https://guides.github.com/introduction/flow/ "Interactive visual Git workflow"
+[GH Help Git setup]: https://help.github.com/articles/set-up-git/ "GitHub Help in Setting up Git"
+[GH Help fork]: https://help.github.com/articles/fork-a-repo/ "GitHub Help on fork a repo"
+[GH Help cache git password]: https://help.github.com/articles/caching-your-github-password-in-git/ "GitHub help on Caching GitHub Password in Git"
+[GH Help webbranch view]: https://help.github.com/articles/viewing-branches-in-your-repository/ "GitHub Help in viewing branches in your repository"
+[GH Help webmerge conflicts]: https://help.github.com/articles/about-merge-conflicts/ "GitHub Help about merge conflicts"
+[GH Help solve webmerge conflicts]: https://help.github.com/articles/resolving-a-merge-conflict-on-github/ "GitHub Help on resolving merge conflicts"
+[GH Help solve git merge conflicts]: https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/ "GitHub Help on resolving a merge conflict using the command line"
+[GH Help GPG]: https://help.github.com/articles/signing-commits-with-gpg/ "GitHub Help on signing commits with GPG"
+
+<!-- Programs and Extensions -->
+[VSC]: https://code.visualstudio.com "Visual Studio Code"
+[Brackets]: http://brackets.io/ "Brackets"
+[Atom]: https://atom.io "Atom"
+[Shiba]: https://github.com/rhysd/Shiba "Shiba"
+[Git]: https://git-scm.com/ "Git"
+[Git Extensions]: https://gitextensions.github.io/ "Git Extensions"
+
+# How to contribute to the osu!wiki
+
+Thanks for your interest in making the osu!wiki better!
+As you can see, everything has been moved to [GitHub][GitHub osu-wiki].
+
+So here are a few things you should have in mind to get you started for contributing to the osu!wiki.
+
+## Disclaimer
+
+This quick-start guide is intended for newcomers not familiar with the feature-branch workflow.
+If you have previous experience in using GitHub, feel free to skip this guide and remember to follow feature-branch workflow for content update.
+
+For complete reference and help with using GitHub, please visit [GitHub Help][GitHub Help].
+
+For a styling guide when writing a page, check out [Article Style Guide][ASG] instead.
+
+If you have any questions, you can inquire at the [osu!dev Discord Server][osu!dev Discord], under ``#osu-wiki`` preferably.
+
+Reading time for the whole page is around an hour.
```

Comment:
Noticed this small thing right here post-merge.


we wouldn't really assume how long it'll take to read the whole thing since everyone has their own pace